pipeline/outputs/loki.md

```diff
@@ -254,12 +255,34 @@ The following configuration:
     structured_metadata pod=$kubernetes['pod_name']
 ```
 
-
 Other common uses for structured metadata include trace and span IDs, process and thread IDs, and log levels.
 
 Structured metadata is officially supported starting with Loki 3.0, and shouldn't be used
 with Loki deployments prior to Loki 3.0.
 
+#### Structured Metadata Maps
+In addition to the `structured_metadata` configuration parameter, a `structured_metadata_map_keys` is available, which can be used to dynamically populate structured metadata from map values in the log record. `structured_metadata_map_keys` can be set with a list of record accessors, where each one should reference map values in the log record. Record accessors which do not match a map value will simply be skipped.  
+
+The following configuration is similar to the above, except now all entries in the log record map value `$kubernetes` will be used as structured metadata entries.
+
+```python
```

Comment:
Can we do the tabs for YAML and TOML config so we have examples of both?

Reply:
https://github.com/fluent/fluent-bit-docs/blob/e9429f39691f30975a0ddb044b2e4cce5793a632/pipeline/outputs/dash0.md?plain=1#L32-L60